README.md

```diff
@@ -43,15 +43,15 @@ Wouldn’t it be nice if the installer wasn’t so large, so you could actually
 
 ## Usage
 
-You can pass a destination filename as a parameter. Please note that if the download and signature checks succeed, the file will be overwritten without asking. If you don’t supply a filename, a random one in your system’s temp directory will be generated.
+You can pass a destination filename as a parameter. Please note that if the download and signature checks succeed, the file will be overwritten without asking. If you don’t supply a filename, `composer.phar` in the current directory will be used.
 
-Whether you supplied a parameter or not, when Tiny Composer Installer succeeds it will echo the destination filename to standard output and return with an exit code of zero. On error, stdout will be empty and a non-zero error code will be returned. This allows you to do something like this:
+Whether you supplied a parameter or not, when Tiny Composer Installer succeeds it will echo the [`realpath()`](http://php.net/manual/en/function.realpath.php) of the destination filename to standard output and return with an exit code of zero. On error, stdout will be empty and a non-zero error code will be returned. This allows you to do something like this:
```

Comment:
Use HTTPs, even though it's "just documentation".

Reply:
You’re right. Fixed.